src/app/query/query-config.ts

```diff
@@ -14,8 +14,14 @@ export type AppUseQueryConfig<QueryFnData, Response> = Pick<
   AllowedReactQueryConfigOptions
 >;
 
+export function isInitializingData<TData, TError = unknown>(
+  query: UseQueryResult<TData, TError>
+): query is QueryObserverSuccessResult<TData, TError> {
+  return query.isFetching || query.isInitialLoading || query.isLoading;
```

Comment:
I believe we want to show skeleton loader only on initial loading?

Reply:
Meant to ask for clarification here, why only on initial loading? If a user clicks on the refetch button, don't we want the loaders to show again, maybe we don't? 🤔 I do have the loading spinner icon showing it is refetching, so maybe that is good enough?

Reply:
if user clicks refetch btn - we want. but we don't want to see skeleton loaders again when we refetch data bc that one in cache became stale

Reply:
`query is QueryObserverSuccessResult<TData, TError>`

Is this the correct type? Why would we expect the success result when the query is still fetching/loading?

Reply:
It appears `isInitialLoading` is being deprecated...

<img width="555" alt="Screenshot 2024-06-03 at 4 22 22 PM" src="https://github.com/leather-wallet/extension/assets/6493321/2fb93bc2-a451-489e-a669-5ad8e552978d">

Reply:
yeah, as I understand `isInitialLoading` and just `isLoading` in now the same in v5?
![Screenshot 2024-06-04 at 09 52 55](https://github.com/leather-wallet/extension/assets/46521087/121f8456-9a29-4d26-a21c-01edfcdbd8d0)